### PR TITLE
Adding return count warnings

### PR DESCRIPTION
**Pull request by mccarthyryanc**
Adding warnings when updating header return count bins are too high for the LAS version. Completely open to changing how the error is caught, and what to call the warning.

### CODE REVIEW

**Comment by mccarthyryanc** [comment]
Sounds good to me! I'll make these changes.

File: pylas/headers/rawheader.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import ctypes
 import datetime
-import warnings
+import logging
 
 from .. import errors
 
+logger = logging.getLogger(__name__)
```
Comment (mccarthyryanc) [inline pylas/headers/rawheader.py]: This isn't exactly what you've done in 'pylas.lib', but I've always preferred seeing the `__name__` when reviewing logs.

Comment (tmontaigu) [reply pylas/headers/rawheader.py]: That's nice, I tried doing the same thing in pylas.lib but it changes the format of the ouput message, I don't really understand why 🤔

**Comment by tmontaigu** [comment]
Nice its much better 👍

**Pull request merged**